<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeacherDialog</class>
 <widget class="QDialog" name="TeacherDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>编号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IndexEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>工号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numberEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>年龄：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ageEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>性别：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sexEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>专业：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="zhuanyeEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>班级：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="banjiEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>联系方式：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phoneEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pswdEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>420</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
